cf_units/build.sh

```diff
@@ -9,7 +9,8 @@ fi
 
 SITECFG=cf_units/etc/site.cfg
 echo "[System]" > $SITECFG
-echo "udunits2_path = $PREFIX/lib/libudunits2.${EXT}" >> $SITECFG
```

Comment:
Raagh, stupid Windows. The difference between these two line is a Win-CRLF (I didn't set the git config option `autocrlf` until _after_ I pushed this :unamused:).